CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+0.1.0 (2016-01-18)
+------
+
+* Update momentum to use Berkshelf and load rake tasks from non-Rails projects ([#19](https://github.com/artsy/momentum/pull/19))
```

Comment:
Could you note the breaking changes here (Berksfile dependency, `cookbooks_install_path`)?

Also, does this assume projects use the latest Chef/Opsworks combo? If so, we should mention that here and in the README.